Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -48,10 +49,6 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed deletion of last composite part raising an exception. [ISXB-804](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-804)
 - Expanded editor and build insight analytics to cover ``.inputactions` asset editor usage, `InputSettings` and common component configurations.
 
-### Changed
-- Changed `DualSenseHIDInputReport` from internal to public visibility
```

Comment:
Why do we remove this?

Reply:
there were two Changed sections by mistake, I deleted one and this line also exists in the remaining section